nautobot/core/tests/test_graphql.py

```diff
@@ -1449,8 +1449,8 @@ def test_query_ip_addresses_filter(self):
                 IPAddress.objects.net_host_contained("10.0.0.0/16").count(),
             ),
             (
-                'prefix: "10.0.2.0/24"',
-                IPAddress.objects.net_host_contained("10.0.2.0/24").count(),
+                'prefix: ["10.0.1.0/24", "10.0.2.0/24"]',
+                IPAddress.objects.net_host_contained("10.0.1.0/24", "10.0.2.0/24").count(),
             ),
```

Comment:
If we can still support single-value filtering, let's keep the original test data here as well.

Reply:
Yup line 1448 uses single value filtering